copy-bitrefill.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
@ahujsak Paul wants us to remove these files, since `copy-plugins.js` should work just fine

Reply:
copy plugins does something different,... it does not move them from the local repository and this is faster than updotting while developing.

Reply:
removed